docs/word/word-add-ins-troubleshooting.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Troubleshooting Word add-ins
+description: Learn how to troubleshoot development errors in Word add-ins.
+ms.date: 02/01/2023
+ms.localizationpriority: medium
+---
+
+# Troubleshooting Word add-ins
+
+This article discusses troubleshooting issues that are unique to Word. Use the feedback tool at the end of the page to suggest other issues that can be added to the article.
+
+## Layout breaks when using `insertHtml` while cursor is in content control in header
+
+This issue may occur in the following case.
```

Comment:
```suggestion
This issue may occur in the following cases.
```
Do all three need to be present in order for this to occur or just one?

Reply:
all three are needed for the scenario - maybe I can say "conditions" instead?

Reply:
"This issue may occur when the following conditions are met."